taxonomies/food/ingredients.txt

```diff
@@ -45437,6 +45478,11 @@ zh: 奇異果
 ciqual_food_code:en: 13021
 ciqual_food_name:en: Kiwi fruit, pulp and seeds, raw
 ciqual_food_name:fr: Kiwi, pulpe et graines, cru
+ecobalyse:en: kiwi-non-eu
+ecobalyse_labels_en_organic:en: kiwi-non-eu
```

Comment:
ecobalyse_labels_en_organic:en: kiwi-non-eu and ecobalyse_labels_en_organic_origins_en_france:en: kiwi-fr should be removed, as they have the same value as ecobalyse_origins_en_france:en: kiwi-fr and ecobalyse_origins_en_european_union:en: kiwi-eu respectively